### PR TITLE
chore(sdk): rename `@prisma/sdk` to `@prisma/internals`

### PR DESCRIPTION
**Pull request by Jolg42**
Related https://github.com/prisma/prisma/issues/10725

TODO rename directory and remaining mentions in next PR

### CODE REVIEW

**Comment by danstarns** [comment]
Will the `/packages/sdk/` directory also be reanamed?

**Comment by Jolg42** [comment]
@danstarns Yes though I think it might be better to do that in another PR after we see things working

**Comment by Jolg42** [comment]
Fails with 
https://github.com/prisma/prisma/runs/6936005555?check_suite_focus=true#step:6:913
```
Error: : packages/cli build: ✘ [ERROR] Could not resolve "@prisma/sdk"
. build: packages/cli build:     ../../node_modules/.pnpm/@prisma+studio-pcw@0.463.0_@prisma+client@packages+client/node_modules/@prisma/studio-pcw/dist/index.es.js:2:198:
. build: packages/cli build:       2 │ ...tryLoadEnvs, getEnvPaths as $aWMXZ$getEnvPaths} from "@prisma/sdk";
. build: packages/cli build:         ╵                                                         ~~~~~~~~~~~~~
. build: packages/cli build:   You can mark the path "@prisma/sdk" as external to exclude it from the bundle, which will remove this error.
Error: : packages/cli build: ✘ [ERROR] [plugin cliLifecyclePlugin] ENOENT: no such file or directory, stat './build/index.js'
```

Will be unblocked after https://github.com/prisma/studio-code/pull/403 is merged

File: packages/cli/package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "@prisma/generator-helper": "workspace:*",
     "@prisma/get-platform": "3.16.0-14.4908cf1dde73491641a0a47fe5f3bebd2430350c",
     "@prisma/migrate": "workspace:*",
-    "@prisma/sdk": "workspace:*",
+    "@prisma/internals": "workspace:*",
     "@prisma/studio": "0.463.0",
     "@prisma/studio-server": "0.463.0",
```
Comment (luanvdw) [inline packages/cli/package.json:77]:
```suggestion
    "@prisma/studio": "0.464.0",
    "@prisma/studio-server": "0.464.0",
```

The latest [version](https://www.npmjs.com/package/@prisma/studio) of Studio has been released.

File: packages/migrate/package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
   },
   "peerDependencies": {
     "@prisma/generator-helper": "*",
-    "@prisma/sdk": "*"
+    "@prisma/internals": "*"
```
Comment (jkomyno) [inline packages/migrate/package.json:43]: This is not strictly related to this PR, but why was `@prisma/sdk` added both as a `devDep` and as a `peerDep`?

File: tsconfig.build.bundle.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
   "compilerOptions": {
     "paths": {
       "@prisma/client": ["./packages/client/src"],
-      "@prisma/sdk": ["./packages/sdk/src"],
-      "@prisma/sdk/dist/*": ["./packages/sdk/src/*"],
+      "@prisma/internals": ["./packages/sdk/src"],
+      "@prisma/internals/dist/*": ["./packages/sdk/src/*"],
```
Comment (jkomyno) [inline tsconfig.build.bundle.json:8]: Uhm, maybe this change invalidates some other comments I highlighted above 🧐

**Review by jkomyno — APPROVED** [review]
Seems fine overall, but please double-check my comments

File: tsconfig.build.bundle.json (hunk 3 above)
Comment (Jolg42) [reply tsconfig.build.bundle.json:8]: Indeed, I think it does (and CI is mostly 🟢 now)

File: packages/migrate/package.json (hunk 2 above)
Comment (Jolg42) [reply packages/migrate/package.json:43]: Both were added on Oct 9, 2019
https://github.com/prisma/prisma/commit/d69cba2aa5fa304bd41aac7a6c57f519433fbbab

Not sure why *both* were added here 🤔
